### PR TITLE
Lint jenkins

### PR DESCRIPTION
**Pull request by Itxaka**
add linting jenkinsfile

requires: https://github.com/SUSE-Cloud/socok8s/pull/330

### CODE REVIEW

**Comment by toabctl** [comment]
@Itxaka you need to add a job template to suse-cloud/automation so we can test this PR directly

**Comment by Itxaka** [comment]
automation PR at https://github.com/SUSE-Cloud/automation/pull/3468

File: Jenkinsfile.lint [hunk 1]
```diff
@@ -0,0 +1,20 @@
+pipeline {
+    options {
+        timestamps()
+        timeout(time: 5, unit: 'MINUTES', activity: true)
+    }
+
+    agent {
+        node {
+            label "cloud-ccp-ci"
```
[the remark this answers is not in the document]
Comment (jhesketh) [reply Jenkinsfile.lint]: We could perhaps also use a more generic node for linting, but it's probably better for us to stay on our resources.

**Comment by Itxaka** [comment]
will send a new PR with the linting as part of the pipeline instead!

**Pull request closed without merging**